src/Plugin/GraphQL/DataProducer/EntityDefinition/Fields.php

```diff
@@ -122,8 +122,8 @@ public function resolve(
     FieldContext $field_context
   ): \Iterator {
     $entity_definition->getBundleEntityType();
-    /** @var \Drupal\Core\Entity\ContentEntityType $value */
-    if ($entity_definition instanceof ContentEntityType) {
+    /** @var \Drupal\Core\Entity\FieldableEntityInterface $entity_definition */
+    if ($entity_definition instanceof FieldableEntityInterface) {
```

Comment:
The entity definition is not the entity, EntityTypeInterface does not implement FieldableEntityInterface.

So I think the check for a ContentEntityType is correct?

Reply:
Maybe 
`    $entity_definition->entityClassImplements(FieldableEntityInterface::class);
`